src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -159,11 +157,14 @@
 
 (def channel-list-component (memoize channel-list-component-fn))
 
-(defn render-page-content [{:keys [name description locked joined id images
-                                   status tokens tags community-color]
+(def request-to-join-text (memoize (fn [is-open?] (if is-open? (i18n/label :t/join-open-community) (i18n/label :t/request-to-join-community)))))
```

Comment:
This line is unreadable to me, could you please break it into multiple lines?

Reply:
Also, not sure memoizing is worth it here, just out of curiosity, have you test the difference in speed? 
(a quick and dirty way, which should give you an idea on whether they are at least comparable is doing something like:
```
(time (dotimes (range 0 10000) (get {:a 1} :a))) // insert inside the operation you care about
```
They both look like map lookups I suppose, so I'd expect both to run in constant time, though I could be wrong

Reply:
You are right, and memoizing what's memoized is no good at all. `i18n/label` expensive calls are already memoized, it's just an overhead to memoize `request-to-join-text`.

Reply:
I will remove the memo 👍